powerline/segments/vim/__init__.py

```diff
@@ -512,6 +512,27 @@ def get_directory(segment_info):
 
 @requires_filesystem_watcher
 @requires_segment_info
+class VimStashSegment(StashSegment):
+	divider_highlight_group = 'stash:divider'
+
+	@staticmethod
+	def get_directory(segment_info):
+		if vim_getbufoption(segment_info, 'buftype'):
+			return None
+		return buffer_name(segment_info)
+
+
+branch = with_docstring(VimStashSegment(),
+'''Return the number of stashes in the current working branch.
+
+Highlight group used: ``stash``.
```

Comment:
Must use `groups`, not `group`. This information is for the linter, not only for the user.